action.yml

```diff
@@ -20,11 +20,13 @@ runs:
   using: composite
   steps:
     - uses: actions/setup-python@v4
+      id: python
```

Comment:
Might it be better to be more baip specific here?

Reply:
Ah you mean rename it to prevent clashes?

Reply:
Yes, I'm assuming this `id` is available after the action runs from the surrounding workflow. So "python-baip" or something like that would be better.